samples/msal-node-samples/custom-INetworkModule-and-network-tracing/HttpClientCurrent.ts

```diff
@@ -255,7 +258,7 @@ const networkRequestViaProxy = <T>(
 };
 
 const networkRequestViaHttps = <T>(
```

Comment:
Would this bug happen if we were using an HttpClient based on Axios?

Reply:
I can confirm that this bug does not occur in the old implementation of HttpClient (axios).

Reply:
I'll be making a new PR adding back the old implementation for cases when proxyUrl isn't used.